app/javascript/App/components/NewsfeedPage/CoinList.js

```diff
@@ -4,6 +4,8 @@ import CoinListItem from './CoinListItem'
 import LoadingIndicator from '../LoadingIndicator'
 import watchlistStarIcon from '../../images/watch-list-star-icon.svg'
 
+const topCoinList = ['BTC']
```

Comment:
this list needs to be dynamic and will be updated when the API returns the top coins